uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -45,21 +48,19 @@ public FileSteps(Platform platform, TestContext testContext) {
     public void arrangeComponentLogFiles(int numFiles, String componentName) throws IOException {
         Path logsDirectory = testContext.installRoot().resolve("logs");
         LOGGER.info("Writing {} log files into {}", numFiles, logsDirectory.toString());
-
         if (!platform.files().exists(logsDirectory)) {
             throw new IllegalStateException("No logs directory");
         }
-
-        if (componentName.equals("aws.greengrass.Nucleus")) {
-            for (int i = 0; i < numFiles; i++) {
-                String fileName = String.format("greengrass_%d.log", i);
-                createFileAndWriteData(logsDirectory, fileName, false);
-            }
-            return;
+        scenarioContext.put(componentName + "LogDirectory",logsDirectory.toString());
+        String filePrefix = "greengrass";
+        if (!componentName.equals("aws.greengrass.Nucleus")) {
+            filePrefix = componentName;
+        }
+        for (int i = 0; i < numFiles; i++) {
+            String fileName = String.format("%s_%d.log", filePrefix,  i);
+            createFileAndWriteData(logsDirectory, fileName, false);
+        }
         }
```

Comment:
Did you compile this? Seems like there is an extra bracket or something funky with the indentation